Sources/SwiftPhoenixClient/Socket.swift

```diff
@@ -36,7 +36,7 @@ public typealias PayloadClosure = () -> Payload?
 
 /// Struct that gathers callbacks assigned to the Socket
 struct StateChangeCallbacks {
-  var open: [(ref: String, callback: Delegated<Void, Void>)] = []
+  var open: [(ref: String, callback: Delegated<(URLResponse?), Void>)] = []
```

Comment:
This doesn't need to be a tuple since it's just a single `response`

```suggestion
  var open: [(ref: String, callback: Delegated<URLResponse?, Void>)] = []
```

Reply:
Most definitely. 👍  

I played with more context passed there, and the tuple just stayed there.

Removed!